nautobot/core/tests/test_views.py

```diff
@@ -41,7 +41,7 @@ def test_get_return_url_explicit_unsafe(self):
         self.assertEqual(self.mixin.get_return_url(request=request, obj=None), reverse("home"))
 
     def test_get_return_url_explicit_punycode(self):
-        request = self.factory.get("/", {"return_url": "/dcım/devices/"})
+        request = self.factory.get("/", {"return_url": "/dcım/devices/"})  # noqa: RUF001
```

Comment:
RUF001 is warning about sneaky unicode characters - here this is fully intentional.